common/FileUtils.java

```diff
@@ -182,4 +182,8 @@ public Message readProtoBinaryUnchecked(String path, Message.Builder builder) {
       throw new RuntimeException(e);
     }
   }
-}
\ No newline at end of file
+
+  public void copyDirectoryToDirectory(String source, String destination) throws IOException {
+    org.apache.commons.io.FileUtils.copyDirectory(new File(source), new File(destination));
```

Comment:
Valerii is working on making everything here use `fileSystem` so that we can use `jimfs` in tests.
If you don't start from `fileSystem`, you end up using the default filesystem which is not the injected one.

You can look at `folderExists()` as an example of how to use it.